### PR TITLE
Add logging configuration to the laravel sample

### PR DESCRIPTION
**Pull request by tmatsuo**

### CODE REVIEW

File: appengine/flexible/laravel/composer.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "require-dev": {
-        "phpunit/phpunit":"~4.8",
+        "google/cloud-logging": "^1.9",
```
Comment (bshaffer) [inline appengine/flexible/laravel/composer.json]: We should also add `google/cloud-error-reporting` for consistency

Comment (tmatsuo) [reply appengine/flexible/laravel/composer.json]: No it's solely for testing. `google/cloud-error-reporting` is not needed for the test.

**Comment by tmatsuo** [comment]
@bshaffer 

Re: README
When you added this directory, apparently you made a decision not to duplicate the same contents in the README as our community tutorial. Let's follow this decision here. Once I update our Stackdriver documentation, I'll update the README with links to those documentation.

**Comment by tmatsuo** [comment]
@bshaffer The DeployTest is passing locally.

**Pull request merged**